core/operation/src/main/java/uk/gov/gchq/gaffer/named/operation/AddNamedOperation.java

```diff
@@ -71,7 +71,7 @@ public class AddNamedOperation implements Operation, Operations<Operation> {
     private AccessPredicate readAccessPredicate;
     private AccessPredicate writeAccessPredicate;
 
-    private static final String CHARSET_NAME = CommonConstants.UTF_8;
+    private static final String CHARSET_NAME = String.valueOf(StandardCharsets.UTF_8);
```

Comment:
This could probably be removed and `StandardCharsets.UTF_8` use directly in place of it. I don't think having this a static String adds anything.